modules/150-user-authn/openapi/config-values.yaml

```diff
@@ -45,7 +45,7 @@ properties:
               kubeconfigGeneratorMasterCA:
                 type: string
                 description: |
-                  If there is an external load balancer in front of the Ingress that terminates HTTPS traffic, then you need to specify the CA of the certificate used on the load balancer so that kubectl can reach the API server.
+                  If there is an external load balancer in front of the Ingress that terminates HTTPS traffic using non-public CA, then you need to specify the CA so it will be included to kubectl-config.
```

Comment:
```suggestion
                  If there is an external load balancer in front of the Ingress that terminates HTTPS traffic using non-public CA, then you need to specify the CA so it will be included in kubectl-config.
```